sermant-plugins/sermant-mq-grayscale/mq-config-common/src/main/java/io/sermant/mq/grayscale/config/MqGrayConfigCache.java

```diff
@@ -48,18 +48,13 @@ public static MqGrayscaleConfig getCacheConfig() {
      * @param eventType eventType
      */
     public static void setCacheConfig(MqGrayscaleConfig config, DynamicConfigEventType eventType) {
-        if (eventType == DynamicConfigEventType.CREATE) {
+        RocketMqConfigUtils.recordTrafficTagsSet(config);
+        RocketMqConfigUtils.updateChangeFlag();
+        if (eventType == DynamicConfigEventType.CREATE || eventType == DynamicConfigEventType.INIT) {
             cacheConfig = config;
-            RocketMqConfigUtils.updateChangeFlag();
-            RocketMqConfigUtils.recordTrafficTagsSet(config);
             return;
         }
-        boolean isAllowRefresh = isAllowRefreshChangeFlag(cacheConfig, config);
-        if (isAllowRefresh) {
-            cacheConfig.updateGrayscaleConfig(config);
-            RocketMqConfigUtils.updateChangeFlag();
-            RocketMqConfigUtils.recordTrafficTagsSet(config);
-        }
+        cacheConfig.updateGrayscaleConfig(config);
     }
```

Comment:
The MODIFY event should also be handled, which indicates config change

Reply:
line 57 is that deal MODIFY event